mudlet-mapper.xml

```diff
@@ -5815,6 +5515,229 @@ end</script>
                     <string>gmcp.Room.Info</string>
                 </eventHandlerList>
             </Script>
+            <Script isActive="yes" isFolder="no">
+                <name>Wings and Bixes</name>
+                <packageName></packageName>
+                <script>local exitsCreated = exitsCreated or {}
+
+local function tempSpecialExit(destination, command)
+  table.insert(exitsCreated, destination)
+  addSpecialExit(mmp.currentroom, destination, command)
+end
+
+function mmp.addWings()
+--Trimmed down version of the original function, meant to only be used for wings. (Where the command varies based on several MCONFIGS)
+  local function getcmd(word)
```

Comment:
The formatting below looks funny, at least in this diff... Maybe consider using `string.format`?